### DISCUSSION TITLE
I am unable to complete installation on Parrot

### DISCUSSION THREAD

**Discussion by ind4skylivey** [polybar/polybar#2811] [Q&A] [answered]
Hello
I have some issues with polybar in Parrot, as i  get the following error

< **Unable to locate package lilbnl-genl-3-devpackage**

Does anyone know how to solve this?

**Comment by patrick96** [comment, marked as answer]
The error message indicates that you are trying to install a package named `libnl-genl-3-devpackage`. However, the name of the package you are looking for is called `libnl-genl-3-dev`.

Also, polybar should be available in the official repos for parrotos

**Reply by ind4skylivey** [reply]
thank u ! 
for the quick answer, i am quite embarrassed to tell this , but it work, looks like i wasn't doing it right.
